homeassistant/components/scsgate.py

```diff
@@ -0,0 +1,154 @@
+"""
+homeassistant.components.scsgate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provides support for SCSGate components.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/scsgate/
+"""
+import logging
+from threading import Lock
+from homeassistant.core import EVENT_HOMEASSISTANT_STOP
+
+from scsgate.connection import Connection
+from scsgate.reactor import Reactor
+from scsgate.tasks import GetStatusTask
+from scsgate.messages import StateMessage, ScenarioTriggeredMessage
+
+DOMAIN = "scsgate"
+
+SCSGATE = None
+_LOGGER = logging.getLogger(__name__)
+
+# pylint: disable=broad-except
```

Comment:
Can you put the disable comment right next to the warning it has to apply . That way you avoid it from accidentally applying to more code than intended.